plugins/skip-button-plugin/src/skip-button.js

```diff
@@ -0,0 +1,74 @@
+import pillarbox from '@srgssr/pillarbox-web';
+
+/**
+ * @ignore
+ * @type {typeof import('video.js/dist/types/button').default}
+ */
+const Button = pillarbox.getComponent('Button');
+
+/**
+ * Represents a SkipButton component for the pillarbox player.
+ *
+ * @class SkipButton
+ */
+class SkipButton extends Button {
+  /**
+   * @type {TextTrackCue}
+   */
+  activeInterval;
+
+  constructor(player, options) {
+    super(player, options);
+    this.addClass('pbw-skip-button', 'vjs-hidden', 'vjs-visible-text');
+    this.player().on('srgssr/interval', ({ data }) => this.handleTimeIntervalChange(data));
+  }
+
```

Comment:
I would add a `dispose` method to remove the event listener when the component is disposed:

```javascript
dispose() {
  this.player().off('srgssr/interval', this.handleTimeIntervalChange);
  super.dispose();
}
```